src/App.tsx

```diff
@@ -1,16 +1,21 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
+const moviesOnPage = [...moviesFromServer];
+
 export const App = () => {
+  const [movies, setMovies] = useState(moviesOnPage);
```

Comment:
why do you need copy, if you just set this array to the state?
```suggestion
export const App = () => {
  const [movies, setMovies] = useState(moviesFromServer);
```